applications/GeoMechanicsApplication/tests/surface_interface_elements/Dirichlet_multi_stage_3_plus_3/ProjectParameters_stage1.json

```
@@ -0,0 +1,140 @@
+        "strategy_type":                      "newton_raphson",
+        "convergence_criterion":              "residual_criterion",
+        "displacement_relative_tolerance":    1.0E-4,
+        "displacement_absolute_tolerance":    1.0E-4,
+        "residual_relative_tolerance":        1.0E-4,
+        "residual_absolute_tolerance":        1.0E-9,
+        "min_iterations":                      6,
+        "max_iterations":                      15,
+        "number_cycles":                      1,
+        "reduction_factor":                   1.0,
+        "increase_factor":                    1.0,
+        "desired_iterations":                 4,
+        "max_radius_factor":                  10.0,
+        "min_radius_factor":                  0.1,
+        "calculate_reactions":                false,
+        "max_line_search_iterations":         5,
+        "first_alpha_value":                  0.5,
+        "second_alpha_value":                 1.0,
+        "min_alpha":                          0.1,
+        "max_alpha":                          2.0,
+        "line_search_tolerance":              0.5,
+        "rotation_dofs":                      false,
+        "linear_solver_settings":             {
+            "solver_type":     "LinearSolversApplication.sparse_lu"
+        },
+        "problem_domain_sub_model_part_list": ["Interfaces"],
+        "processes_sub_model_part_list":      ["FixedNodes","AllNodes","SideZX"],
+        "body_domain_sub_model_part_list":    ["Interfaces"]
+    },
+    "output_processes": {
+        "gid_output": [{
+            "python_module": "gid_output_process",
+            "kratos_module": "KratosMultiphysics",
+            "process_name": "GiDOutputProcess",
+            "Parameters":    {
+                "model_part_name": "PorousDomain.porous_computational_model_part",
+                "output_name": "Dirichlet_multi_stage_3_plus_3_stage1",
+                "postprocess_parameters": {
+                    "result_file_configuration": {
+                        "gidpost_flags":       {
+                            "WriteDeformedMeshFlag": "WriteUndeformed",
+                            "WriteConditionsFlag":   "WriteElementsOnly",
+                            "GiDPostMode":           "GiD_PostAscii",
+                            "MultiFileFlag":         "SingleFile"
+                        },
+                        "file_label":          "step",
+                        "output_control_type": "step",
+                        "output_interval":    1,
+                        "body_output":         true,
+                        "node_output":         true,
+                        "skin_output":         false,
+                        "plane_output":        [],
+                        "nodal_results":       ["DISPLACEMENT"],
+                        "gauss_point_results": []
+                    },
+                    "point_data_configuration":  []
```

Empty list can be removed
```suggestion
                    }
```